### DISCUSSION TITLE
Stop event propagation for dialog trigger

### DISCUSSION THREAD

**Discussion by arkreddy21** [shadcn-ui/ui#1485] [Q&A]
I'm using the Dialog trigger button inside a div that has another onClick functionality. How do I stop the click event propagating to the parent when I clicked on the dialog trigger button? I know about event.stopPropagation() but I don't think it's usable in this context as there's no onClick function for dialog trigger that I can see

**Comment by arkreddy21** [comment]
Okay, I can just add `onClick={(e) => e.stopPropagation()}` to the trigger button. I was just worried that it may override default click functionality.

**Reply by stivenm16** [reply]
This solution does not work if you also support mobile
